docs/courses/csintro3/orientation/math.md

```diff
@@ -1,101 +1,171 @@
 # Math
 
-table with math blocks -> equivalent
+When transitioning into JavaScript, we see that doing calculations and writing math formulas becomes a lot easier.
 
-| blocks | JavaScript |
-| -- | -- |
-| x  |  * |
-| +  |  + | 
-| %  |  / | 
+## Math Operators
 
-etc
+When writing code with blocks, we saw that we needed to have a block for each math operator (e.g. +, -, &times;, &divide;). This can be pretty annoying and make our code formulas more complicated than they need to be. We'll see that in JavaScript, formulas are a lot more condensed.
+
+Here is how we translate each operator from blocks to JavaScript
+
+|Operation 					|Block					|Javascript			|
+|:---						|:---:					|:---				|
```

Comment:
Are you using 3 space tabs or something by any chance? PXT uses [4 space tabs](https://github.com/jwunderl/pxt-arcade/blob/b9340533fdf9f34ed894eecf793bf0c7f79e8d72/.clang-format#L2), so these tables look really wonky on here / ideally should match as it gets hard to read when they're spaced out inconsistently